.github/workflows/scala_container_publishing.yml

```diff
@@ -53,6 +54,13 @@ jobs:
         with:
           name: goatrodeo-${{ github.run_id }}
           path: target/scala-*/goatrodeo*.jar
+
+      - name: Release
+        uses: softprops/action-gh-release@v2
+        with:
+          files: |
+             target/scala-*/goatrodeo*.jar
+          make_latest: true  
```

Comment:
I'd rather have the release be the trigger;  can we add files to the release from the workflow that results rather than like this? It's a much more structured way to trigger things, and we can reject ones that haven't run a build yet.

Reply:
Release tag as trigger also means we get to manually intervene in the release to write good release notes, which always requires a human touch since it's comms and just listing commits isn't that useful sometimes.